assets/src/components/editing/elements/youtube/YoutubeElement.tsx

```diff
@@ -47,10 +50,14 @@ export type YouTubeProps = EditorProps<ContentModel.YouTube>;
 export const YouTubeEditor = (props: YouTubeProps) => {
   const focused = useFocused();
   const selected = useSelected();
+  const { model } = props;
 
-  const parameters = 'disablekb=1&modestbranding=1&showinfo=0&rel=0&controls=0';
-  const fullSrc =
-    'https://www.youtube.com/embed/' + (props.model.src || CUTE_OTTERS) + '?' + parameters;
+  useCommandTargetable(
```

Comment:
When I did the video-player command buttons, I had come up with this system to allow more components to be registered and added as command targets, was glad it wasn't wasted effort in the end.